setup.py

```diff
@@ -25,7 +25,7 @@
     ],
     python_requires=">=3.7",
     install_requires=[
-        "pandas==0.25.2",
+        "pandas>=0.25.2",
```

Comment:
Are there any null-type handling parts where we need to be aware of the `NA` type change in Pandas 1?

Reply:
This PR runs all SDK tests against both specified versions of pandas (0.25.2 and 1.0.0), so there's nothing we're aware of yet.

Reply:
tox ftw